lightkurve/tests/test_targetpixelfile.py

```diff
@@ -624,3 +624,16 @@ def test_get_header():
     # ``tpf.header`` is deprecated
     with pytest.warns(LightkurveDeprecationWarning, match='deprecated'):
         tpf.header
+
+def test_plot_pixels():
+    tpfs = [KeplerTargetPixelFile(filename_tpf_one_center), TessTargetPixelFile(filename_tpf_one_center)]
```

Comment:
The statement `TessTargetPixelFile(filename_tpf_one_center)` will yield this warning during the tests:
```LightkurveWarning: A Kepler data product is being opened using the `TessTargetPixelFile` class.```

While the warning could be silenced, it would be fine to simply run this test just for `KeplerTargetPixelFile(filename_tpf_one_center)`.  That will get rid of the warning without reducing the test coverage in any significant way.